module/zfs/arc.c

```diff
@@ -4996,6 +4998,14 @@ arc_reap_cb_check(void *arg, zthr_t *zthr)
 		arc_no_grow = B_FALSE;
 	}
 
+	/*
+	 * this function should be called unconditional
+	 * every 60 seconds, so we need to use this small little
+	 * quirk to avoid a independend thread.
+	 */
```

Comment:
How about:

```c
        /*
         * Called unconditionally every 60 seconds to reclaim unused
         * zstd compression and decompression contexts.  This is done
         * here to avoid the need for an independent thread.
         */
```

Reply:
Yeah, "should be" is not the right wording indeed.

Reply:
done